buttons/src/main/res/values/dimens.xml

```diff
@@ -1,16 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <dimen name="vtmn_button_insettop">0dp</dimen>
+    <dimen name="vtmn_button_insetbottom">0dp</dimen>
 
-    <dimen name="vtmnButtonHorizontalPaddingMedium">24dp</dimen>
-    <dimen name="vtmnButtonVerticalPaddingMedium">14dp</dimen>
-    <dimen name="vtmnButtonIconStartPaddingMedium">24dp</dimen>
-    <dimen name="vtmnButtonIconEndPaddingMedium">8dp</dimen>
-    <dimen name="vtmnButtonIconSizeMedium">20dp</dimen>
+    <dimen name="vtmn_button_horizontalpadding_medium">24dp</dimen>
```

Comment:
If we add the module name, we should add a `s` at the end for all properties here.